src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,20 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
+    private static final int SIZE = 6;
 
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
Don't extend your `Main/Application` class from `FigureSupplier` or `ColorSupplier`. Instead, create an instance of the class as you are doing.